docs/README.Ubuntu.md

```diff
@@ -147,6 +148,28 @@ sudo apt install doxygen libcap-dev libsndio-dev libmariadbd-dev
 
 **[back to top](#table-of-contents)**
 
+### 3.3 Ubuntu <= 18.04
+Ubuntu 18.04 and lower ship an outdated [Meson](https://mesonbuild.com/) and [nasm](https://nasm.us/) package. 
+In order to compile Kodi with AV1 support, you have to manually update both Meson and nasm.
+
+> This is a requirement by the [dav1d](https://code.videolan.org/videolan/dav1d) AV1 decoder used in ffmpeg.
+
+#### Meson
+```
+pip3 install --user meson
+```
+
+Make sure `~/.local/bin` is in your PATH.
+
+Verify your Meson version by running `meson -v`. The version displayed should be >= 0.47.0.
+
+#### nasm (x86_64 / amd64)
+```
+wget https://mirrors.edge.kernel.org/ubuntu/pool/universe/n/nasm/nasm_2.14-1_amd64.deb && sudo apt install ./nasm_2.14-1_amd64.deb
+```
+
+Verify your nasm version by running `nasm -v`. The version displayed should be >= 2.14.
```

Comment:
In case you wondered. dav1d requires nasm 2.14 since this (recent) commit: https://code.videolan.org/videolan/dav1d/-/commit/447b7c638ddc4b414d06fbe06b78392cc1005c0b